### DISCUSSION TITLE
source .env does not working

### DISCUSSION THREAD

**Discussion by panditdhamdhere** [Cyfrin/foundry-full-course-cu#1498] [Q&A]
hey folks after adding my rpc url in my .env file I'm running `source .env` command and it returns ` source .env
SEPOLIA_RPC_URL: command not found`
Whats the problem?

**Comment by ArnaudBand** [comment]
Hello @panditdhamdhere,

The error occurs when there are spaces around the equal sign in your `.env` file. For example:

```.env
SEPOLIA_RPC_URL = 888775443243555
```

When you run `source .env`, the shell interprets the line as a command because of the space before the equal sign, resulting in the error `SEPOLIA_RPC_URL: command not found`.

To fix this, remove the spaces around the equal sign in your `.env` file:

```.env
SEPOLIA_RPC_URL=888775443243555
```

After making this change, running `source .env` should work as expected.

**Reply by cdw3lch** [reply]
I had the same issue. Thank you, this resolved it.

**Reply by 14himanshi** [reply]
I was stuck on this for a long time, This resolved my issue as well . Thank you :)

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.

**Comment by EngrPips** [comment]
Hello @panditdhamdhere , did you resolve this issue already ?

**Reply by panditdhamdhere** [reply]
@EngrPips yes sir

**Reply by EngrPips** [reply]
Awesome, good to hear. Enjoy
